File: benchmarks/arraysum/arraysum.c
int sum_array() {
    int arr[10000];
    for (int i = 0; i < 10000; i++) arr[i] = i;
    int total = 0;
    for (int iter = 0; iter < 1000; iter++)
        for (int i = 0; i < 10000; i++) total += arr[i];
    return total;
}
int main() { sum_array(); return 0; }

File: benchmarks/arraysum/arraysum.lua
function sum_array()
    local arr = {}
    for i = 0, 9999 do arr[i] = i end
    local total = 0
    for iter = 0, 999 do
        for i = 0, 9999 do total = total + arr[i] end
    end
    return total
end
sum_array()
